public/app/features/alerting/TestRuleResult.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { PureComponent } from 'react';
+import { LoadingPlaceholder, JSONFormatter } from '@grafana/ui';
+
 import appEvents from 'app/core/app_events';
 import { CopyToClipboard } from 'app/core/components/CopyToClipboard/CopyToClipboard';
-import { getBackendSrv } from '@grafana/runtime';
 import { DashboardModel } from '../dashboard/state/DashboardModel';
-import { LoadingPlaceholder, JSONFormatter } from '@grafana/ui';
+import { getBackendSrv, BackendSrv } from '../../core/services/backend_srv';
```

Comment:
Why not importing from `@grafana/runtime`?

Reply:
I couldn't mock the `getBackendSrv` from `@grafana/runtime`because the reference each other somehow. But I can take another look at it if `@grafana/runtime` is more correct.

Reply:
No, it's not more correct - it's more about giving a way for plugins to access internal services. But would be good to use the same APIs when possible. Apparently when mocking internal backages dependencies you need to use full path. See my commit f60c350b6f177d218c62365604553f8ff1199604

Reply:
Awesome thanks!